src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carRent = 40;
+  let fullCost = days * carRent;
```

Comment:
The variable name `carRent` could be more descriptive. Consider renaming it to `DAILY_RENTAL_RATE` to clearly represent its purpose.